### PR TITLE
Added solution to this exercise

### PR DESCRIPTION
**Pull request by b0c0de**
- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]
LGTM 👍

File: curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.english.md [hunk 1]
```diff
@@ -90,7 +90,33 @@ tests:
 <section id='solution'>
 
 ```js
```
Comment (RandellDawson) [inline curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.english.md]:
```suggestion
```html
```

Comment (ojeytonwilliams) [reply curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.english.md]: Good catch. Also, it's weird that it works.

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Please make the changes @RandellDawson suggested

**Comment by RandellDawson** [comment]
@thecodingaviator I made the change I suggested on the user's behalf.  If you are ok with the change, please merge this PR.

Thank you.

**Review by thecodingaviator — APPROVED** [review]

**Comment by thecodingaviator** [comment]
Thanks for contributing to the freeCodeCamp repository. Congratulations on your first PR! 🎊 

Look forward to more contributions from you! ✈️

**Pull request merged**